serving/docker/transformers-neuronx.Dockerfile

```diff
@@ -0,0 +1,58 @@
+# -*- mode: dockerfile -*-
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file
+# except in compliance with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+FROM ubuntu:20.04
+ARG djl_version=0.20.0~SNAPSHOT
+ARG python_version=3.8
+EXPOSE 8080
+
+# Sets up Path for Neuron tools
+ENV PATH="/opt/aws/neuron/bin:${PATH}"
+
+COPY dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
+WORKDIR /opt/djl
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
+ENV OMP_NUM_THREADS=1
+ENV MODEL_SERVER_HOME=/opt/djl
+ENV JAVA_OPTS="-Xmx1g -Xms1g -Xss2m -XX:-UseContainerSupport -XX:+ExitOnOutOfMemoryError"
+
+ENTRYPOINT ["/usr/local/bin/dockerd-entrypoint.sh"]
+CMD ["serve"]
+
+COPY scripts scripts/
+RUN mkdir -p /opt/djl/conf && \
+    mkdir -p /opt/djl/deps
+COPY config.properties /opt/djl/conf/
+RUN mkdir -p /opt/djl/bin && cp scripts/telemetry.sh /opt/djl/bin && \
+    echo "${djl_version} inf2" > /opt/djl/bin/telemetry && \
+    scripts/install_python.sh ${python_version} && \
+    scripts/install_djl_serving.sh $djl_version && \
+    scripts/install_inferentia2.sh && \
+    python3 -m pip install torch-neuronx \
```

Comment:
Please make sure all important packages have `ARG` version. So we can make sure container stable when we cut a release